site/static/shacl-extensions.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Eclipse RDF4J SHACL Extensions</title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
```

Comment:
Do we need a CQ for this?

Reply:
Good point - I think we do. Actually worth verifying that it's acceptable before investing heavily. I doubt it's a problem, but W3C use a non-standard license for their software, so best be sure that the legal team are ok with it.

Reply:
I have changed my mind on this actually. We are not distributing the respec code itself as part of our repo or as part of our project downloads, and from what I can tell of similar things in IPZilla (e.g. a CQ for a hugo theme that was withdrawn on advice of the IP team), they consider this kind of scenario low-risk enough that IP review is not necessary.